operators/OWNERS

```diff
@@ -1,3 +1,4 @@
 approvers:
   - knkski
   - rfmvasconcelos
+  - DomFleischmann
```

Comment:
Can you make it in alphabetical order please?

Reply:
Damn, I was about to merge this! Nice catch.

Reply:
Should we remove myself to stick to 2 people? As I'm more on the docs these days?

Reply:
@RFMVasconcelos Answered here: https://github.com/kubeflow/katib/pull/1426#issuecomment-775230424.
It's fine to have more than 2 people.

Reply:
Awesome! :)